src/main/target/common_pre.h

```diff
@@ -237,6 +237,8 @@ extern uint8_t _dmaram_end__;
 #define USE_SERIAL_PASSTHROUGH
 #define USE_GYRO_REGISTER_DUMP  // Adds gyroregisters command to cli to dump configured register values
 #define USE_IMU_CALC
+
+#if (!defined(CLOUD_BUILD))
```

Comment:
Rather than sprinkling multiple such sections through this file, would it not be simpler to have a single block of `#undef` statements at the end if `CLOUD_BUILD` is defined? That would make it easier to maintain a set of features which are controlled by the cloud build.

Reply:
This is true, but having undefs does make it somewhat more obscure. I am not fussed either way.

Reply:
I just think it’s easier to group all this into a single block. Happy to approve as is if you’d prefer to go that way.